kernel/environ.ml

```diff
@@ -488,7 +488,7 @@ let add_universes_set ~strict ctx g =
 let push_context_set ?(strict=false) ctx env =
   map_universes (add_universes_set ~strict ctx) env
 
-let push_qualities ctx env =
+let unsafe_push_qualities ctx env =
```

Comment:
I'm not convinced by the naming convention, most functions in Environ do not check whether the binding already exists yet without it being written on the label.

Reply:
this change isn't actually needed for this PR so I'll remove it